### PR TITLE
{bio}[GCC/9.3.0] vcflib v1.0.2 + deps

### PR DESCRIPTION
**Pull request by jfgrimm**
(created using `eb --new-pr`)
typo in original commit msg - require(d) ~~#12837~~ ~~#12838~~ ~~#12839~~

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 7 out of 8 (5 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/8155190c979a19619e90f5ed374b1038 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node031.pri.viking.alces.network - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz, Python 2.7.5
See https://gist.github.com/4ed134adf714aa5f4ea973de0df4f31e for a full test report.

File: easybuild/easyconfigs/f/fsom/fsom-20141119_fix-abs-overload.patch [hunk 1]
```diff
@@ -0,0 +1,16 @@
+diff -Nru fsom-a6ef318fbd347c53189384aef7f670c0e6ce89a3/fsom.c fsom-a6ef318fbd347c53189384aef7f670c0e6ce89a3.2/fsom.c
```
Comment (branfosj) [inline easybuild/easyconfigs/f/fsom/fsom-20141119_fix-abs-overload.patch:1]: Please add description and author to this patch.

File: easybuild/easyconfigs/s/smithwaterman/smithwaterman-20160702_build-shared-lib.patch [hunk 2]
```diff
@@ -0,0 +1,125 @@
+diff -Nru smithwaterman-2610e259611ae4cde8f03c72499d28f03f6d38a7.orig/Makefile smithwaterman-2610e259611ae4cde8f03c72499d28f03f6d38a7/Makefile
```
Comment (branfosj) [inline easybuild/easyconfigs/s/smithwaterman/smithwaterman-20160702_build-shared-lib.patch:1]: Please add description and author to this patch.

File: easybuild/easyconfigs/v/vcflib/vcflib-1.0.2_use-external-deps.patch [hunk 3]
```diff
@@ -0,0 +1,108 @@
+diff -Nru vcflib.orig/CMakeLists.txt vcflib/CMakeLists.txt
```
Comment (branfosj) [inline easybuild/easyconfigs/v/vcflib/vcflib-1.0.2_use-external-deps.patch:1]: Please add description and author to this patch.

File: easybuild/easyconfigs/v/vcflib/vcflib-1.0.2-GCC-9.3.0-Python-3.8.2.eb [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+sources = ['v%(version)s.tar.gz']
+patches = ['vcflib-1.0.2_use-external-deps.patch']
+checksums = [
+    'a3ef230864328d96eb3b086bf5bbcc34d6699710c358444621f6c3e3f507c910',  # v1.0.2.tar.gz
+    '8f46f06b42d2a88377ae5e666a023353d0171cbe70b6b3b4289c0bd7f387ac6c',  # vcflib-1.0.2_use-external-deps.patch
+]
+
+builddependencies = [
+    ('binutils', '2.34'),
+    ('CMake', '3.16.4'),
```
Comment (branfosj) [inline easybuild/easyconfigs/v/vcflib/vcflib-1.0.2-GCC-9.3.0-Python-3.8.2.eb:26]: Please add `pkg-config` (0.29.2) as a build dependency.

**Review by branfosj — CHANGES_REQUESTED** [review]
If there are suitable items for `sanity_check_commands` then it would be good to add some checks on the executables.

**Comment by jfgrimm** [comment]
@branfosj thanks for taking a look! I've added author/desc to the patches that were missing it, and have added some `sanity_check_commands`.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/d0f934269cef83686350ba1c5a0fa217 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/f2ba4047ed8e49b4929721afeb9be793 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @jfgrimm!

**Pull request merged**